proposals/wg-reference-types.md

```diff
@@ -0,0 +1,86 @@
+
+## Purpose
+
+As cloud native development continues to grow, there is increased interest in evolving registries to natively store, discover, and pull a graph of content associated with specific container images in a registry. Use cases for said associated artifacts include but are not limited to Software Bill of Materials (SBoM), security scan results, and signing. Having a native way to store and discover artifacts associated with other artifacts enables end-users to answer the question of: “What SBOMs or signatures are associated with this container image?”
+
+## Scope
+
+* Define and deliver the capability to store, discover, and pull a graph of artifacts associated with a specific artifacts to OCI distribution compliant registries. These set of capabilities has been commonly known as "reference types" or "references".
+  * Define supported use cases
+  * Document impact to existing user-facing tools and registries
+  * Define the method for creating, distributing, and discovering referenced objects
+  * Document user expectations for promoting an artifact between registries with its references
+  * Document onboarding process for registries and user-facing tools to adopt reference types
+  * Defined expectations for artifact reference lifecycle management
+  * Deliver a reference implementation of the reference types proposal
+
+## Out of Scope
+
+* Identified out of scope items will be listed here as WG progresses
+
+## Intended work product
+
+* Referrers API specification that provides the ability to discover references to existing container images. These include listing signatures, SBoMs, security scan results that refer to the digest of a manifest. The referrers API specification will sit within or along side the Distribution specification.
+* Identify, and document the pros and cons for versioning the existing manifests, compared with creating a new manifest to support reference types.
+
+## Proposed Owners
+
+* Lachlan Evenson (@lachie83)
+* Justin Cormack (@justincormack)
+* Michael Brown (@michaelb990)
+* Derek McGowan (@dmcgowan)
+* Jon Johnson (@jonjohnsonjr)
+
+## Stakeholders
+
+* Microsoft
+* AWS
+* Docker
+
+## Related issues/PRs
+
+* https://github.com/opencontainers/tob/issues/96
+* https://github.com/opencontainers/artifacts/pull/27
+* https://github.com/opencontainers/artifacts/pull/29
+* https://github.com/opencontainers/artifacts/pull/37
+* https://github.com/opencontainers/image-spec/issues/827
+* https://github.com/opencontainers/image-spec/pull/828
+* https://github.com/oras-project/artifacts-spec/
```

Comment:
Just a note here that this repository is not itself an OCI Project even though many of the people working on it are also involved in OCI (and in this proposed WG).